docs/minio_installation_macOS.md

```diff
@@ -0,0 +1,35 @@
+# Minio Installation on macOS
```

Comment:
We should rename the file name as a `homebrew.md`  which would provide more insights on why we choose this and link this with README.md `macOS` installation section.